README.md

```diff
@@ -44,9 +44,12 @@ To build Prometheus from the source code yourself you need to have a working
 Go environment with [version 1.5 or greater installed](http://golang.org/doc/install).
 
 You can directly use the `go` tool to download and install the `prometheus`
-and `promtool` binaries into your `GOPATH`:
+and `promtool` binaries into your `GOPATH`. We use Go 1.5's experimental
+vendoring feature, so you will also need to set the `GO15VENDOREXPERIMENT=1`
+environment variable in this case:
 
-    $ go get github.com/prometheus/prometheus/...
+    $ export GO15VENDOREXPERIMENT=1
+    $ go get github.com/prometheus/prometheus/cmd/...
```

Comment:
Why not in one line without polluting the local env?

`GO15VENDOREXPERIMENT=1 go get github.com/prometheus/prometheus/cmd/...`